app/models.py

```diff
@@ -0,0 +1,230 @@
+from peewee import *
```

Comment:
I was thinking maybe changing this import to be explicit, this way we can reduce our variable pollution, thoughts?

Reply:
I expected the import of models in the main app to be an explicit `import models` which won't pollute our variable space, but an explicit import here, instead, would work just fine.